less/cards.less

```diff
@@ -443,7 +462,7 @@
           .parent-infop {
             width: 20%;
             &.parent-infop-first {
-              width: 70px;
```

Comment:
Could you explain where this applies? I couldn't find.

Reply:
in list format they permit to show 2 activity  and the number of other activity

Reply:
Isn't it the goal of line 245/259?